docs/getting-started.md

```diff
@@ -0,0 +1,59 @@
+# Getting started
+
+## Prerequisites
+
+You'll need the following tools installed in order to run the project locally:
+
+-   Node.js v12+
+-   MySQL v5.7+
+-   [AWS CLI](http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html) configured on your machine
+
+Once you have the prerequisites installed on your machine, and have the project checked out locally, run these commands in your terminal of choice:
```

Comment:
Worth adding something in here about asking the team for AWS permissions otherwise the secrets step below will fail?

Reply:
Good call, added a line right next to the command for fetching secrets.